pytorch/buildspec-1-8-1.yml

```diff
@@ -68,6 +69,34 @@ images:
                          *DEVICE_TYPE ]
     context:
       <<: *TRAINING_CONTEXT
+  BuildSageMakerCPUPTTrainPy3DockerImage:
```

Comment:
remove `BuildSageMakerCPUPTTrainPy3DockerImage` and `BuildSageMakerGPUPTTrainPy3DockerImage` configuration since this is not needed for PT1.8.1